transport/transport.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name                     = 'transport'
-    spec.version                  = '2.5.2'
+    spec.version                  = '2.5.3'
```

Comment:
normally we should not touch .podspec file if it is not in scope of the release process.